src/FileServiceManager/FileService/Startup.cs

```diff
@@ -0,0 +1,94 @@
+using Microsoft.AspNetCore.Builder;
+using Microsoft.AspNetCore.Hosting;
+using Microsoft.AspNetCore.Localization;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Hosting;
+using Microsoft.OpenApi.Models;
+using System.Collections.Generic;
+using System.Globalization;
+using System.Linq;
+using FileService.Helpers;
+using FileService.DAL;
+
+namespace FileServiceManager.FileService
+{
+    public class Startup
+    {
+        public Startup(IConfiguration configuration)
+        {
+            Configuration = configuration;
+        }
+
+        public IConfiguration Configuration { get; }
+
+        // This method gets called by the runtime. Use this method to add services to the container.
+        public void ConfigureServices(IServiceCollection services)
+        {
+            services.AddControllers();
+            services.AddSwaggerGen(options =>
+            {
+                options.IgnoreObsoleteActions();
+                options.DescribeAllParametersInCamelCase();
+                options.SwaggerDoc("v1", new OpenApiInfo
+                {
+                    Title = "Travely.FileService API",
+                    Version = "v1",
+                    TermsOfService = new System.Uri("https://github.com/titancamp/travely"),
+                    Contact = new OpenApiContact { Name = "https://www.servicetitan.com" }
+                });
+                //options.AddSecurityDefinition("JWT", new OpenApiSecurityScheme
```

Comment:
do not leave unused code in the committed code

Reply:
Fixed